Colab-TextGen-GPU.ipynb

```diff
@@ -5,7 +5,8 @@
     "colab": {
       "private_outputs": true,
       "provenance": [],
-      "gpuType": "T4"
+      "gpuType": "T4",
+      "include_colab_link": true
```

Comment:
What does this do exactly?

Reply:
I thought it would add a link to google colab so someone could just click to go to colab from the notebook but does not seem to have worked as expected.